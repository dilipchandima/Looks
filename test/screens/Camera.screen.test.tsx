import React from 'react';

import {render} from '@testing-library/react-native';
import CameraScreen from 'screens/Camera.screen';

const nav = {
  navigate: jest.fn(),
};

describe('Camera Screen', () => {
  it('should match to snapshot', () => {
    const screen = render(<CameraScreen navigation={nav} />);
    expect(screen).toMatchSnapshot();
  });
});
