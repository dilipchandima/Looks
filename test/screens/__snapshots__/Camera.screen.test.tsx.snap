// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Camera Screen should match to snapshot 1`] = `
<View
  style={
    Object {
      "backgroundColor": "#000000",
      "flex": 1,
      "paddingTop": 100,
    }
  }
>
  <RCTSafeAreaView
    style={
      Object {
        "flex": 1,
      }
    }
  >
    <View
      style={
        Object {
          "flex": 1,
        }
      }
    >
      <LooksCamera
        onResultImageExported={[Function]}
        style={
          Object {
            "borderRadius": 20,
            "height": 1000,
            "width": 750,
          }
        }
      />
      <View
        style={
          Object {
            "alignItems": "center",
            "flex": 1,
            "justifyContent": "center",
          }
        }
      >
        <View
          style={
            Object {
              "alignItems": "center",
              "borderColor": "#FFFFFF",
              "borderRadius": 35,
              "borderWidth": 2,
              "height": 70,
              "justifyContent": "center",
              "width": 70,
            }
          }
        >
          <View
            accessible={true}
            collapsable={false}
            focusable={true}
            onClick={[Function]}
            onResponderGrant={[Function]}
            onResponderMove={[Function]}
            onResponderRelease={[Function]}
            onResponderTerminate={[Function]}
            onResponderTerminationRequest={[Function]}
            onStartShouldSetResponder={[Function]}
            style={
              Object {
                "backgroundColor": "#FFFFFF",
                "borderRadius": 30,
                "height": 60,
                "opacity": 1,
                "width": 60,
              }
            }
          />
        </View>
      </View>
    </View>
  </RCTSafeAreaView>
</View>
`;
